services/ml/ml_utils_dml.cc

```diff
@@ -12,9 +12,30 @@
 #include "services/ml/dml/float16_compressor.h"
 #include "services/ml/dml/format_data.h"
 #include "services/ml/dml_d3dx12_utils.h"
+#include "services/ml/public/mojom/constants.mojom.h"
 
 namespace ml {
 
+namespace {
+
+DML_TENSOR_DATA_TYPE TensorDataType(int32_t type) {
+  switch (type) {
+    case mojom::TENSOR_FLOAT32:
+      return g_support_f16 ? DML_TENSOR_DATA_TYPE_FLOAT16
+                           : DML_TENSOR_DATA_TYPE_FLOAT32;
+    case mojom::TENSOR_INT32:
+      // BUGBUG: should be DML_TENSOR_DATA_TYPE_INT32, but reduce output tensor
+      // need UINT32.
+      return DML_TENSOR_DATA_TYPE_UINT32;
```

Comment:
It would be better to handle this case just for argmax op.

Reply:
Thanks done.